src/init.js

```diff
@@ -87,133 +92,178 @@ export const init = async (options) => {
+				owner: scaffold.owner,
+				repo: scaffold.repo,
+				path: 'snapfu.config.yml',
+			});
 
-			// scaffold reference
-			const scaffold = scaffolds[answers2.scaffold];
-			if (!scaffold) {
-				console.log(chalk.red(`Failed to find the selected scaffold ${answers2.scaffold}...\n`));
+			try {
+				const buffer = new Buffer.from(contentResponse.data.content, 'base64');
+				const fileContents = buffer.toString('ascii');
+				scaffold.advanced = YAML.parse(fileContents);
+			} catch (err) {
+				console.log(chalk.red(`Failed to parse snapfu.config.yml contents...\n Please report this issue to Athos support.`));
 				exit(1);
 			}
+		} catch {
+			// do nothing - no advanced configuration available
+		}
 
-			try {
-				const contentResponse = await octokit.rest.repos.getContent({
-					owner: scaffold.owner,
-					repo: scaffold.repo,
-					path: 'snapfu.config.yml',
-				});
-
-				try {
-					const buffer = new Buffer.from(contentResponse.data.content, 'base64');
-					const fileContents = buffer.toString('ascii');
-					scaffold.advanced = YAML.parse(fileContents);
-				} catch (err) {
-					console.log(chalk.red(`Failed to parse snapfu.config.yml contents...\n`));
-					exit(1);
+		// ask follow up scaffold questions (if they exist)
+		if (scaffold.advanced?.variables?.length) {
+			let advancedQuestions = [];
+			scaffold.advanced.variables.forEach((variable) => {
+				if (variable.name && variable.type && variable.message) {
+					// remove the value
+					delete variable.value;
+					advancedQuestions.push(variable);
 				}
-			} catch (err) {
-				if (err.status !== 404) {
-					console.log(chalk.red(`Failed to fetch snapfu.config.yml...\n`));
-					exit(1);
-				}
-			}
+			});
 
-			// ask additional questions (for advanced scaffolds)
-			if (scaffold.advanced?.variables?.length) {
-				let advancedQuestions = [];
-				scaffold.advanced.variables.forEach((variable) => {
-					if (variable.name && variable.type && variable.message) {
-						// remove the value
-						delete variable.value;
-						advancedQuestions.push(variable);
-					}
-				});
+			scaffold.answers = await inquirer.prompt(advancedQuestions);
+		}
 
-				scaffold.answers = await inquirer.prompt(advancedQuestions);
-			}
+		// ask for siteId (as this is used as a snapfu variable when copying over scaffold)
+		const questions3 = [
+			{
+				type: 'input',
+				name: 'siteId',
+				message: 'Please enter the siteId as found in the SMC console (a1b2c3):',
+				validate: (input) => {
+					return input && input.length > 0 && /^[0-9a-z]{6}$/.test(input);
+				},
+			},
+		];
+		const answers3 = await inquirer.prompt(questions3);
+
+		let useGitHubRepo = false;
+		let repositoryAnswers = {};
+
+		const githubQuestions = [
+			{
+				type: 'confirm',
+				name: 'useGitHubRepo',
+				message: 'Would you like to create a GitHub repository? (requires login)',
+				when: () => {
+					return isLoggedIn;
+				},
+				default: useGitHubRepo,
```

Comment:
I think the default here should be `true` - as I could see this being a pain point for implementations teams.